elastic/kibana
# [Fleet] Disable ILM policy in serverless

## Summary

Disable ILM policy installation in serverless mode.


For this I introduced a new config flag 
```
xpack.fleet.internal.ILMPoliciesDisabled: true
```

## How to test

Add this to your kibana config file and install a package with ILM policies (synthetics) , then check no ILM policies are installed.
```
xpack.fleet.internal.ILMPoliciesDisabled: true
```


## TODO

- [x] add manual test instructions
